pycbc/waveform/bank.py

```diff
@@ -670,20 +683,19 @@ def __getitem__(self, index):
 
         htilde = htilde.astype(self.dtype)
         htilde.f_lower = self.f_lower
```

Comment:
Looks like this must be changed to `htilde.f_lower = f_low`, otherwise `self.f_lower` is None and I get an exception later on:
```
Traceback (most recent call last):
  File "/home/tito/sw_install/pycbc_latest/bin/pycbc_inspiral", line 4, in <module>
    __import__('pkg_resources').run_script('PyCBC==4046c9', 'pycbc_inspiral')
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 534, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1438, in run_script
    execfile(script_filename, namespace, namespace)
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-4046c9-py2.7.egg/EGG-INFO/scripts/pycbc_inspiral", line 305, in <module>
    sigmasq=template.sigmasq(segments[0].psd))
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-4046c9-py2.7.egg/pycbc/waveform/bank.py", line 59, in sigma_cached
    self.approximant, psd, len(psd), psd.delta_f, self.f_lower)
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-4046c9-py2.7.egg/pycbc/waveform/waveform.py", line 824, in get_waveform_filter_norm
    return _filter_norms[approximant](psd, length, delta_f, f_lower)
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-4046c9-py2.7.egg/pycbc/waveform/spa_tmplt.py", line 127, in spa_tmplt_norm
    k_min = int(f_lower / delta_f)
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'
```